utils/add-comment.js

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+
+const postComment = (articleId, comment, username) => {
+  return axios
+    .post(
+      `https://nc-news-backend-project-u3fr.onrender.com/api/articles/${articleId}/comments`,
+      { username: username, body: comment }
+    )
+    .then((response) => {
+      return response.data;
+    });
+};
+
```

Comment:
The downside of having a separate file for each api request function is you can't make good use of axios' `baseURL` functionality. If you had all of them in one file you could use an axios instance and specify the baseURL ('https://nc-news-backend-project-u3fr.onrender.com') in one place. This would mean if your API url ever changed you would only need to update one line of code, rather than an in multiple places in each of your utils files 📁